mitodl/bootcamp-ecommerce
# User klass REST API

#### What are the relevant tickets?
closes of #6 

#### What's this PR do?
Adds REST API for listing the klasses where the user is allowed to pay and the list of all the payments done

#### How should this be manually tested?
There are 2 endpoint to test (the detail and the list for klass info).
By default the `BootcampAdmissionClient` now returns an authorization for any local klass in the database, so just create a couple of klasses and check that they show up in the rest API
